packages/components/src/Utils/callback.test.js

```diff
@@ -5,7 +5,8 @@ describe('JsEvent', () => {
     AlteryxLanguageCode: 'en',
     Gui: {
       SetConfiguration: jest.fn(),
-      GetConfiguration: jest.fn()
+      GetConfiguration: jest.fn(),
+      Callbacks: {}
```

Comment:
Callbacks should be inside `window.Alteryx`, not inside the `Gui` obj. [This](https://git.alteryx.com/tempo2/gen-ai/gen-ai-designer-tools/-/blob/main/ui/custom.d.ts#L54) is what Im using for reference. Also, do the other properties need to be included here? Not sure how what the logic is to determine what gets included or not

Reply:
I kept getting an error about not having callbacks in Gui, I think possible Callbacks is a field in both